src/features/ToDos/ToDoItem/ToDoItem.module.css

```diff
@@ -0,0 +1,37 @@
+.toDoText {
+  display: inline-block;
+  margin-left: 1rem;
+  color: #a3a5b2;
+}
+
+.crossed {
+  text-decoration: line-through;
+  color: #36d7b7;
+}
+
+.isCompletedCheckBox {
+  width: 1.3em;
+  height: 1.3em;
+  border-radius: 60%;
+  background-color: white;
+  vertical-align: middle;
+  border: 1px solid #ddd;
+  appearance: none;
+  outline: none;
```

Comment:
Do not remove outline without adding additional styling on `:focus`, because using navigation via keyboard it is impossible to understand which element currently selected. It reduces accessibility